File: jsNBA/scripts/_check.mjs
import fs from 'fs';
const h = JSON.parse(fs.readFileSync('data/history.json','utf8'));
const runs = h.runs || [];
console.log('Last 5 NBA run dates:');
for (const r of runs.slice(-5)) {
  console.log('  ', r.dateDisplay || r.date);
}
console.log('Total runs:', runs.length);

File: pyNBA/scripts/_check.py
# scripts/_check.py
import json

def main():
    with open("data/history.json", "r") as f:
        h = json.load(f)
    runs = h.get("runs", [])
    print("Last 5 NBA run dates:")
    for r in runs[-5:]:
        print("  ", r.get("dateDisplay") or r.get("date"))
    print("Total runs:", len(runs))

if __name__ == "__main__":
    main()
